### PR TITLE
Create macalifa.yml

### PR DESCRIPTION
**Pull request by thecodrr**
I would like to add my project to `up-for-grabs.net`

### CODE REVIEW

File: _data/projects/macalifa.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: Macalifa
+desc: A music player for UWP and Windows 10.
+site: https://github.com/theweavrs/Macalifa
+tags:
+- .NET
```
Comment (shiftkey) [inline _data/projects/macalifa.yml]: There's a issue reported in the CI tests - this should be in quote marks.

**Review by shiftkey — CHANGES_REQUESTED** [review]

File: _data/projects/macalifa.yml (hunk 1 above)
Comment (thecodrr) [reply _data/projects/macalifa.yml]: I added the quote marks for .NET, .NET Core and C# tag. Is that okay?

**Review by shiftkey — APPROVED** [review]

File: _data/projects/macalifa.yml (hunk 1 above)
Comment (shiftkey) [reply _data/projects/macalifa.yml]: @theweavrs :thumbsup:

Comment (thecodrr) [reply _data/projects/macalifa.yml]: Thank you!

**Pull request merged**